libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Preds/CertState.hs

```diff
@@ -286,12 +330,19 @@ mainD seed = defaultMain $ testIO "Testing DState Stage" (demoD Interactive seed
 
 -- ===============================================
 
+tell :: String -> (Subst era -> a) -> (Subst era -> a)
+tell s f (Subst m) =
+  case Map.lookup "drepState" m of
+    Nothing -> trace (s ++ " NO") (f (Subst m))
+    Just x -> trace (s ++ " YES " ++ show x) (f (Subst m))
+
```

Comment:
Let's snap a warning on it to prevent it actually being used somewhere in future PRs
```suggestion
{-# WARNING tell "This function suppose to be used for debugging only" #-}
```

Reply:
I removed it